packages/babel-plugin-transform-es2015-parameters/src/rest.js

```diff
@@ -155,21 +159,18 @@ function optimiseIndexGetter(path, argsId, offset) {
     index = t.binaryExpression("+", path.parent.property, t.numericLiteral(offset));
   }
 
-  path.parentPath.replaceWith(loadRest({
+  path.parentPath.replaceWith(restIndex({
     ARGUMENTS: argsId,
     INDEX: index,
   }));
 }
 
 function optimiseLengthGetter(path, argsLengthExpression, argsId, offset) {
```

Comment:
We can remove this since it's not used anymore argsLengthExpression (and it's references)?